test/e2e/etcdcluster_test.go

```diff
@@ -38,8 +38,9 @@ import (
 	"tkestack.io/kstone/test/fixtures"
 )
 
-var _ = ginkgo.Describe("kstone-etcd-operator etcdcluster", getGinkgoFunc(fixtures.DefaultKstoneEtcdOperatorClusterName, fixtures.DefaultKstoneEtcdOperatorPodName))
 var _ = ginkgo.Describe("imported etcdcluster", getGinkgoFunc(fixtures.DefaultImportedClusterName, fixtures.DefaultImportedPodName))
+var _ = ginkgo.Describe("kstone-etcd-operator etcdcluster", getGinkgoFunc(fixtures.DefaultKstoneEtcdOperatorClusterName, fixtures.DefaultKstoneEtcdOperatorPodName))
```

Comment:
DefaultKstoneEtcdOperatorClusterName -> DefaultHttpKstoneEtcdOperatorClusterName
DefaultKstoneEtcdOperatorPodName -> DefaultHttpKstoneEtcdOperatorPodName
It is better to unify the naming with the following https kstone-etcd-operator cluster.